From 5c5d6b715a5531e2742969526b1571d39608e01c Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 16 Aug 2016 14:23:23 +0300
Subject: [PATCH 0416/1345] net: mvpp2x: change read type in
 mv_pp2x_rxq_received

commit  cead9e3fc775f1b17bdbbad779ca41a15fe1fc9f from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- change read call function from relaxed_read to regular read

Change-Id: Ie90394e3c70d6a7a7f0d1a32d355b000d61684f5
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31974
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Yuval Caduri <cyuval@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw.h b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw.h
index c451e55..b6169aa 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw.h
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_hw.h
@@ -126,7 +126,7 @@ static inline int mv_pp2x_txq_phys(int port, int txq)
 /* Get number of Rx descriptors occupied by received packets */
 static inline int mv_pp2x_rxq_received(struct mv_pp2x_port *port, int rxq_id)
 {
-	u32 val = mv_pp2x_relaxed_read(&port->priv->hw, MVPP2_RXQ_STATUS_REG(rxq_id));
+	u32 val = mv_pp2x_read(&port->priv->hw, MVPP2_RXQ_STATUS_REG(rxq_id));
 
 	return val & MVPP2_RXQ_OCCUPIED_MASK;
 }
-- 
1.7.9.5

